recipe/meta.yaml

```diff
@@ -23,14 +31,24 @@ requirements:
         - hdf5
 
 test:
+    requires:
+        - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
Linter is complaining about this. We can ignore the complaint.

Though won't this be dealt with correctly as HDF5 is a build/run dependency, @patricksnape?